packages/@aws-cdk/aws-codebuild/test/test.codebuild.ts

```diff
@@ -479,6 +480,758 @@ export = {
+                }
+              ]
+            }
+          },
+          "MyProjectCodeBuildPolicy1074D15A": {
+            "Type": "AWS::IAM::Policy",
+            "Properties": {
+              "PolicyDocument": {
+                "Statement": [
+                  {
+                    "Action": [
+                      "ec2:CreateNetworkInterface",
+                      "ec2:DescribeNetworkInterfaces",
+                      "ec2:DeleteNetworkInterface",
+                      "ec2:DescribeSubnets",
+                      "ec2:DescribeSecurityGroups",
+                      "ec2:DescribeDhcpOptions",
+                      "ec2:DescribeVpcs",
+                      "ec2:CreateNetworkInterfacePermission"
+                    ],
+                    "Effect": "Allow",
+                    "Resource": "*"
+                  }
+                ],
+                "Version": "2012-10-17"
+              },
+              "PolicyName": "CodeBuildEC2Policy",
+              "Roles": [
+                {
+                  "Ref": "MyProjectRole9BBE5233"
+                }
+              ]
+            }
+          },
+          "MyProject39F7B0AE": {
+            "Type": "AWS::CodeBuild::Project",
+            "Properties": {
+              "Artifacts": {
+                "Type": "NO_ARTIFACTS"
+              },
+              "Environment": {
+                "ComputeType": "BUILD_GENERAL1_SMALL",
+                "Image": "aws/codebuild/ubuntu-base:14.04",
+                "PrivilegedMode": false,
+                "Type": "LINUX_CONTAINER"
+              },
+              "ServiceRole": {
+                "Fn::GetAtt": [
+                  "MyProjectRole9BBE5233",
+                  "Arn"
+                ]
+              },
+              "Source": {
+                "Location": {
+                  "Fn::Join": [
+                    "",
+                    [
+                      {
+                        "Ref": "MyBucketF68F3FF0"
+                      },
+                      "/path/to/source.zip"
+                    ]
+                  ]
+                },
+                "Type": "S3"
+              },
+              "VpcConfig": {
+                "SecurityGroupIds": [
+                  {
+                    "Fn::GetAtt": [
+                      "SecurityGroup1F554B36F",
+                      "GroupId"
+                    ]
+                  }
+                ],
+                "Subnets": [
+                  {
+                    "Ref": "MyVPCPrivateSubnet1Subnet641543F4"
+                  },
+                  {
+                    "Ref": "MyVPCPrivateSubnet2SubnetA420D3F0"
+                  },
+                  {
+                    "Ref": "MyVPCPrivateSubnet3SubnetE1B8B1B4"
+                  }
+                ],
+                "VpcId": {
+                  "Ref": "MyVPCAFB07A31"
+                }
+              }
+            }
+          }
+        }
+      });
+      test.done();
+    }
   },
 
```

Comment:
Missing tests for the error conditions (when `securityGroups` or `allowAllOutbound` are provided without `vpc`).

Reply:
Will resolve this week